### DISCUSSION TITLE
Request the ability to silence background audio

### DISCUSSION THREAD

**Discussion by CircularKen** [edrlab/thorium-reader#1778] [Ideas]
The support for 'ambient' or background audio, whilst not technically an EPUB standard (using ibooks:soundtrack), is very much appreciated in Thorium. It is well implemented as it loops and it continues to play uninterrupted and without restarting when moving between pages that share the same audio track. I agree with the decision to have it switched on by default.

Logging this as an issue because, as I'm sure you know, people will want an option to be able to switch this off.

The option to silence the background audio should be independent to other 'foreground' audio including media overlays.

**Comment by danielweck** [comment]
Good idea.
